cli/args/flags.rs

```diff
@@ -1092,6 +1092,10 @@ Show documentation for runtime built-ins:
     )
     .arg(import_map_arg())
     .arg(reload_arg())
+    .arg(lock_arg())
+    .arg(no_lock_arg())
+    .arg(no_npm_arg())
+    .arg(no_remote_arg())
```

Comment:
Working in this flags.rs is pretty terrible because you need to update two places instead of one to set these. At the very least the set and parse code for each sub command should be right next to each other instead of being spread out in the file. We should improve this in the future.

Reply:
I think we could group them as `lock_args()` and pass them using `.args()`